assets/css/main.css

```diff
@@ -2378,23 +2378,23 @@
 		width: 100%;
 	}
 
-		.features li {
+		.features > li {
```

Comment:
Asked in the other PR before: what is the `>` doing?

Reply:
It's a selector for all `li` elements that are direct descendants of elements with the `features` class